app/src/main/res/values/donottranslate_questionnaire_values.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools">
```

Comment:
I'm not sure. I'm just listing all options to strings.xml for @gen0083 . Since @gen0083 need this to implement View, I guess. (I cannot work this until today's late night. so I sent PR)